client/python/openlineage/client/transport/transport.py

```diff
@@ -0,0 +1,20 @@
+from openlineage.client.run import RunEvent
+
+
+class Config:
+    @classmethod
+    def from_dict(cls, params: dict):
+        return cls()
+
+
+class Transport:
```

Comment:
Add some doc comments. Pointers to the factory and existing impls would be helpful